tasks/opsep/001-update-l2pao-to-aliased-l1pao/SignFromJson.s.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+import {SignFromJson as OriginalSignFromJson} from "script/SignFromJson.s.sol";
+import {Predeploys} from "@eth-optimism-bedrock/src/libraries/Predeploys.sol";
+import {AddressAliasHelper} from "@eth-optimism-bedrock/src/vendor/AddressAliasHelper.sol";
+import {console2 as console} from "forge-std/console2.sol";
+import {Vm, VmSafe} from "forge-std/Vm.sol";
+import {LibString} from "solady/utils/LibString.sol";
+import {GnosisSafe} from "safe-contracts/GnosisSafe.sol";
+
+interface IProxyAdmin {
+    function owner() external view returns (address);
+}
+
+contract SignFromJson is OriginalSignFromJson {
+    using LibString for string;
+
+    // Chains for this task.
+    string constant l1ChainName = "sepolia";
+    string constant l2ChainName = "op";
+
+    // Safe contract for this task.
+    GnosisSafe l2paoSafe =
+        GnosisSafe(payable(0xb41890910b05dCba3d3dEF19B27E886C4Ab406EB));
+
+    IProxyAdmin l2pa = IProxyAdmin(Predeploys.PROXY_ADMIN);
+    address constant unaliasedL1PAO =
+        0x1Eb2fFc903729a0F03966B917003800b145F56E2; // Safe
+
+    function setUp() public {}
+
+    function checkL2PA() internal {
+        console.log("Running assertions on the L2PA");
+        address l2paOwner = l2pa.owner();
+        require(
+            l2paOwner == AddressAliasHelper.applyL1ToL2Alias(unaliasedL1PAO),
+            "checkL2PA-100"
+        );
+
+        address payable l1Address = payable(
+            AddressAliasHelper.undoL1ToL2Alias(l2paOwner)
+        );
+        uint256 originalFork = vm.activeFork();
+        vm.createSelectFork(vm.envString("L1_ETH_RPC_URL")); // begins on the latest block (can't use block.number as it's the wrong chain here).
```

Comment:
another nit lol, just to clarify intent since we wouldn't want a function of `block.number` here (unlike other scripts)
```suggestion
        vm.createSelectFork(vm.envString("L1_ETH_RPC_URL")); // Forks on the latest block.
```